README.md

```diff
@@ -1,8 +1,42 @@
-# codenames
-This game is often a four player game where the players are split into two teams. These teams compete to guess their words first.
-
-### Setup
-- A 5x5 "board" of cards is laid down for all players to see.
-- Each team nominates a codegiver and a guesser.
-- The codegiver for each team is given a seperate board which shows them which card is their team's, which are the other team's, which are neither teams, and finally the one card that results in immediate loss.
-- The team that goes first will have one more card than the other team.
+# Codenames
+Codenames is a board/card game where two teams, red and blue, compete against each to be the first to guess all of their team's cards. This project uses a TUI to implement the setup of the board and the game play.
+
+<img src="doc/codenames.gif" width="75%"/>
+
+## Learning Objectives
+- Familiarity with Python
+- Learning to use Object Oriented Programming with Python
+- Structuring a game loop
+- Implementing pytest
+- Understanding the process of refactoring code to make it more readable and easy to debug
+
+## Running the Game
+- `python3 main.py`
```

Comment:
For the `Running the Game` and `Testing the Game` sections, use [multiline code blocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) with `sh` syntax highlighting.